library/Zend/Mvc/View/Console/DefaultRenderingStrategy.php

```diff
@@ -42,7 +42,7 @@ public function render(MvcEvent $e)
         // marshal arguments
         $response  = $e->getResponse();
 
-        if (empty($result)) {
+        if (empty($result) || $result === true) {
```

Comment:
When you expect the method `hasChildren()` to be there, isn't it better to test if `$result` contains the class with that method?
```php
if (! $result instancof ConsoleViewModel) {
```